packages/material-ui-system/src/createTheme/createTheme.d.ts

```diff
@@ -0,0 +1,42 @@
+import { Breakpoints, BreakpointsOptions } from './createBreakpoints';
+import { Mixins, MixinsOptions } from './createMixins';
+import { Shadows } from './shadows';
+import { Shape, ShapeOptions } from './shape';
+import { Spacing, SpacingOptions } from './createSpacing';
+import { Transitions, TransitionsOptions } from './createTransitions';
+
+export { Breakpoint, BreakpointOverrides } from './createBreakpoints';
+
+export type Direction = 'ltr' | 'rtl';
+
```

Comment:
This is the important part to focus on when reviewing. Is this structure enough for a baseline theme? Should we include here the `components`, `typography`, `zIndex` although we don't provide any defaults?

Reply:
In my opinion, we should add the components here, the thing is then, later on, we will need to augment from the `@material-ui/system` when adding new components. For example https://next.material-ui.com/customization/styled/ (I believe it should still be fine).